requirements.txt

```diff
@@ -1,7 +1,7 @@
 adafruit-blinka==8.12.0
 adafruit-circuitpython-bno055
 luma.oled==3.12.0
-pillow==10.0.0
+pillow
```

Comment:
I've just tested version 9.5.0 on the hardware and emulated version.  I'll set this in the requirements file